### PR TITLE
commited Test2 branch

### PR DESCRIPTION
**Pull request by drift92**

### CODE REVIEW

**Comment by nisswarn** [comment]
Hey nisswarn, code looks good. plz merge.

**Comment by drift92** [comment]
OK. Thanks from drift

File: skills.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ Jenkins
 Git
 Ansible
 Docker
+drift92 is used to test PULL request
```
Comment (nisswarn) [inline skills.txt:5]: good job

**Review by nisswarn — APPROVED** [review]

**Comment by nisswarn** [comment]
@drift please merge

**Pull request merged**